tools/events.php

```diff
@@ -142,8 +141,7 @@ protected function list(): void
      */
     protected function clear(): void
     {
-        $cacheManager = CacheManager::getManager();
-        $cacheManager->flush('event');
+        PKPEventServiceProvider::clearCache();
```

Comment:
Likely a general problem, the cache can't be forcefully cleared from the administration area. I guess there was already discussion somewhere, right?

Reply:
Now I remembered that I've talked about it on a task to retire the `CacheManager`, but it ended up forgotten. Here's a new fresh issue: https://github.com/pkp/pkp-lib/issues/8922 😅